ssg/build_derivatives.py

```diff
@@ -108,6 +108,17 @@ def remove_idents(tree_root, namespace, prod="RHEL"):
                 fix.text = re.sub(r"CCE-[0-9]*-[0-9]*", "", fix.text)
 
 
+def remove_cce_reference(tree_root, namespace, prod="RHEL"):
```

Comment:
The `prod` argument is not used in the function, so what about removing it?

Reply:
Removed it.